qemu-shutdown

```diff
@@ -1,7 +1,23 @@
 #!/bin/bash
 
+# TODO: Remove this when shutdown-deferrer is fully implemented and returns
+# proper defer status.
 sleep 60
 
+POLL_INTERVAL=5
+POLL_TIMEOUT=600
+
+# Poll shutdown-deferrer service in order to wait for proper node draining
+# before VM shutdown.
+DEFER="yes"
+while [ "$DEFER" != "no" -a "$DEFER" != "false" -a $POLL_TIMEOUT -gt 0 ]
```

Comment:
`shutdown-deferrer` currently returns `no`, but based on suggestion from @kopiczko I might change this to `false` which is more suitable towards computers.

Reply:
Lets have variables for the values so we can more easily change them later.